afl/Makefile

```diff
@@ -0,0 +1,11 @@
+#	$NetBSD$
+
+KMOD=	kcov_afl
+SRCS=	kcov_afl.c
+
+$(error uncomment clone and setup debugcon_printf as below)
+# CPPFLAGS+=	-I/~/workspace/debugcon_printf/
```

Comment:
I propose to add a debug symbol `-DHAVE_DEBUGCON_PRINTF` or similar and wrap with it `#include <debugcon_printf.h>` etc.

Reply:
fixed